sdcm/cluster.py

```diff
@@ -1871,7 +1871,10 @@ def install_package(self,
                         package_version: str = None,
                         ignore_status: bool = False) -> None:
         if self.distro.is_rhel_like:
-            pkg_cmd = 'yum'
+            if not self.parent_cluster.cluster_backend == "docker":
```

Comment:
@fruch shouldn't we also adjust code for k8s to support rhel? (it's not just this one, so can be done in followup)

Reply:
It can be a followup

Anyhow we should minimize the usage of it as much as possible